### ISSUE TITLE
Review undocumented features

### ISSUE THREAD

**Issue by rlafuente** [shoebot/shoebot#215]
There are some features we have that are not present in the docs (or maybe I overlooked those):

[ ] Drawbot support -- how to use
[ ] Choosing backend via environment variable
[ ] `show()`, `finish()` and other useful internals should be properly explained

I'll update this list as I find more, feel free to add as well

**Comment by stuaxo** [comment]
The thing about drawbot is that it didn't stop evolving.

Drawbot as it exists now is a pretty different beast to the one we support.

**Comment by rlafuente** [comment]
You're totally right there, our Drawbot support was implemented a long time ago (long before its resurgence).

However, I never got around to document how to use it, which would be at least minimally useful. (and from there, maybe figure out a roadmap so that we can support modern Drawbot features)

**Comment by stuaxo** [comment]
I was going to make the gfx backend chosable by commandline, but ran out of motivation.

There is an example in the code of the env var usage at the top of backend.py

https://github.com/shoebot/shoebot/blob/master/shoebot/core/backend.py
